README.md

```diff
@@ -1,18 +1,17 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# New York Times Article
 
-> One paragraph statement about the project.
+>  Clone a New York Times Article
 
 ![screenshot](./app_screenshot.png)
 
 Additional description about the project and its features.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML & CSS
+- Visual Studio Code
 
 ## Live Demo
```

Comment:
- [ ] Include a live link to the page. You can use this [`tool`](https://raw.githack.com/) to generate a live version of the page for you.